broken-site/broken-site-impl/src/main/java/com/duckduckgo/brokensite/impl/BrokenSitePromptRCFeature.kt

```diff
@@ -16,6 +16,61 @@
 
 package com.duckduckgo.brokensite.impl
 
+import com.duckduckgo.anvil.annotations.ContributesRemoteFeature
+import com.duckduckgo.app.di.AppCoroutineScope
+import com.duckduckgo.common.utils.DispatcherProvider
+import com.duckduckgo.di.scopes.AppScope
+import com.duckduckgo.feature.toggles.api.FeatureSettings
+import com.duckduckgo.feature.toggles.api.RemoteFeatureStoreNamed
+import com.duckduckgo.feature.toggles.api.Toggle
+import com.duckduckgo.feature.toggles.api.Toggle.DefaultValue
+import com.squareup.anvil.annotations.ContributesBinding
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
+import dagger.SingleInstanceIn
+import javax.inject.Inject
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+
+@ContributesRemoteFeature(
+    scope = AppScope::class,
+    featureName = "brokenSitePrompt",
+    settingsStore = BrokenSitePromptRCFeatureStore::class,
+)
 interface BrokenSitePromptRCFeature {
     // TODO (cbarreiro) Implement broken site prompt
+    @DefaultValue(false)
+    fun self(): Toggle
+}
+
+@ContributesBinding(AppScope::class)
+@SingleInstanceIn(AppScope::class)
+@RemoteFeatureStoreNamed(BrokenSitePromptRCFeature::class)
+class BrokenSitePromptRCFeatureStore @Inject constructor(
+    private val repository: BrokenSiteReportRepository,
+    @AppCoroutineScope private val appCoroutineScope: CoroutineScope,
+    private val dispatcherProvider: DispatcherProvider,
+) : FeatureSettings.Store {
+
+    private val jsonAdapter by lazy { buildJsonAdapter() }
+
+    override fun store(jsonString: String) {
+        appCoroutineScope.launch(dispatcherProvider.io()) {
```

Comment:
nit: no need to specify a dispatcher here

Reply:
Done